Connect4.py

```diff
@@ -1,7 +1,92 @@
 GRID_WIDTH = 7
 GRID_HEIGHT = 6
 
-grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+class Board:
+
+    def __init__(self):
+        self.grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+
+    def draw(self):
+        print("\n  0    1    2    3    4    5    6")
+        for row in self.grid:
+            print(row)
+
+    def drop(self, col):
+        col = int(col)
+        for i in range(5, -1, -1):
+            if self.grid[i][col] == "_":
+                self.grid[i][col] = playerLetter(player)
+                break
+
+    def win(self, player):
+        for i in range(7):
+            if self.check_col(i, player):
+                return True
+        for i in range(6):
+            if self.check_row(i, player):
+                return True
+        #diagonals
+        row = 3
+        while row <= 5:
+            for i in range(0, 4):
+                if self.check_slash(i, row):
+                    return True
+            row += 1
+        row = 3
+        while row <= 5:
+            for i in range(6, 2, -1):
+                if self.check_backslash(i, row):
+                    return True
+            row += 1
+        return False
+
+    def check_col(self, col, player):
+        x = int(col)
```

Comment:
convert outside, add comments that specify the contract of your function if necessary (e.g. check_col checks if 4 tokens of type player in a row exist in column 'col'. 
params:
   col - integer - the column to check.
   player - whatever